cmake/Codegen.cmake

```diff
@@ -76,11 +76,21 @@ function(GEN_XPU file_yaml)
     --per-operator-headers
     --static-dispatch-backend
     --backend-whitelist=XPU
+    # --xpu: generate in-tree RegisterXPU.cpp for in-tree OPs
```

Comment:
We will have two `RegisterXPU.cpp` ?

Reply:
@fengyuan14  Yes, in stock pytorch, we've already supported structured kernel generation for GEMM ops. The PR https://github.com/pytorch/pytorch/pull/139025 has been landed.